[wip] (#9116)
odoo/documentation: content/administration/on_premise/source.rst

         .. tab:: Debian/Ubuntu

            For Debian-based systems, the packages are listed in the `debian/control

>             <{GITHUB_PATH}/debian/control>`_ file of the Odoo sources.

Reviewer: It could be nice to keep the href so one can review the list of packages from their browser.

            .. code-block:: console

>                $ cd odoo #CommunityPath

Reviewer: The *CommunityPath* placeholder is explained above and presented in the `/CommunityPath` format in commands in many other places on this page; let's keep it.
```suggestion
               $ cd /CommunityPath
```
It could maybe be made clearer, but then it should be done for the entire page at once, not just this one command block.

               $ sudo ./setup/debinstall.sh

>             The `setup/debinstall.sh` script will parse the `debian/control` and install found

Reviewer: ```suggestion
            The `setup/debinstall.sh` script will parse the `debian/control` file and install the found
```

            packages.

         .. tab:: Install with pip

            .. warning::

> 
>                Using pip may lead to security issue and broken dependencies, only do this if you
>                know what you rare doing.

Reviewer: - removed extra line
- issue**s**
- **,** -> **;**
- rare -> are
```suggestion
               Using pip may lead to security issues and broken dependencies; only do this if you
               know what you are doing.
```
